metadata.rb

```diff
@@ -7,7 +7,7 @@
 source_url 'https://github.com/sous-chefs/mariadb' if respond_to?(:source_url)
 issues_url 'https://github.com/sous-chefs/mariadb/issues' if respond_to?(:issues_url)
 chef_version '>= 12.6' if respond_to?(:chef_version)
-version '1.5.3'
+version '1.5.4'
```

Comment:
Can we roll this change back? I'd like to get some testing improvements in first.